src/test/java/stepDefinition/restul/UserSteps.java

```diff
@@ -0,0 +1,128 @@
+import static util.ScenarioContext.*;
+
+public class UserSteps extends AbstractStepDef {
+    private Store store;
+    private List<UserInputFields> foundUserInputFields;
+
+    @Given("User is on main home page")
+    public void user_is_on_main_home_page() {
+        isDisplayed(paraBankHomePage.getParaBankLogoOnTheHomePage());
+        highlightElement(paraBankHomePage.getParaBankLogoOnTheHomePage(), driver);
+    }
+
+    @When("User clicks on 'Register' button")
+    public void user_is_clicks_on_Register_button() throws InterruptedException {
+        click(paraBankHomePage.getRegisterButtonOnHomePage());
+        assertEquals("Signing up is easy!", registerPage.getSighUpIsEasyPhraseOnRegistPage().getText());
+    }
+
+    @When("{} and {} is inserted")
+    public void password_and_username_is_inserted(String password, String username) {
+        highlightElement(registerPage.getUserName(), driver);
+        highlightElement(registerPage.getPassword(), driver);
+        highlightElement(registerPage.getConfirmedPassword(), driver);
+        saveData(USERNAME, username);
+        saveData(PASSWORD, password);
+        sendKey(registerPage.getUserName(), username);
+        sendKey(registerPage.getPassword(), password);
+        sendKey(registerPage.getConfirmedPassword(), password);
+    }
+
+    @When("User inserts the few fields")
+    public void user_inserts_the_few_fields(DataTable table) throws InterruptedException {
+        store = new Store();
+        foundUserInputFields = new ArrayList<>();
+        List<Map<String, String>> rows = table.asMaps(String.class, String.class);
+
+        for (Map<String, String> columns : rows) {
+            store.addInputFieldsValue(new UserInputFields(columns.get("lastName"), columns.get("address"), columns.get("city"), columns.get("state"),
+                    columns.get("zipCode"), columns.get("phone"), columns.get("ssn")));
+            highlightElement(registerPage.getFirstName(), driver);
+            highlightElement(registerPage.getLastName(), driver);
+            highlightElement(registerPage.getAddress(), driver);
+            highlightElement(registerPage.getCity(), driver);
+            highlightElement(registerPage.getState(), driver);
+            highlightElement(registerPage.getZipCode(), driver);
+            highlightElement(registerPage.getPhone(), driver);
+            highlightElement(registerPage.getSsn(), driver);
+            String userName = getData(USERNAME).toString();
+            sendKey(registerPage.getFirstName(), userName);
+            sendKey(registerPage.getLastName(), columns.get("lastName"));
+            sendKey(registerPage.getAddress(), columns.get("address"));
+            click(registerPage.getRegisterButton());
+            assertEquals("Signing up is easy!", registerPage.getSighUpIsEasyPhraseOnRegistPage().getText());
+        }
+    }
+
+    @When("Not all mandatory fields are inserted error appears")
```

Comment:
please rename the step according with the functionality that are checked and the Gherkin language